data_diff/database.py

```diff
@@ -22,6 +22,24 @@ def parse_table_name(t):
     return tuple(t.split("."))
 
 
+def import_helper(package: str = None, text=""):
```

Comment:
Maybe we should add the extra name to the list of database tables in the README? Or at least in the `How to install` example, list all of them?

Reply:
I think it's the same issue: make it easier for people to figure out what drivers to install and when